kubernetes/crds.yaml

```diff
@@ -0,0 +1,357 @@
+# The set of CRDs used by machine-controller-manager
```

Comment:
Well, this controller runs as an OOT controller, I needed those crds to be applied to the test cluster, otherwise the controller will complain about missing crds. To create machines based on machine-classes per se you don't need to run the MCM and you can use this OOT controller alone. However, if you would like to create machine-deployment, then yes the MCM should be there. In other words, if someone would like to run this controller only( for testing e.g) then you need to apply those crds, and having them here would allow the tester to ignore MCM completely. 

Disclaimer: I have no problem of actually moving them, but having integration tests in the future for this project only would need those CRDS.

Reply:
I don't find them in the vsphere OOT controller, and I don't like the duplication much. I would prefer to have them removed and direct developers to their proper location if they are needed. When integration tests are added, we could decide to readd them back if this is the best option.

Reply:
I don't like duplication either tbh. As you said, let's get rid of them for now and we think about adding them later.